### PR TITLE
WIP - STS-2144: Added forms not supported in v1 content creation error

### PR DESCRIPTION
**Pull request by kivra-thohut**
Work In Progress - Do not merge

### CODE REVIEW

**Comment by RSBRK** [comment]
Why not set it as a draft PR?

**Review by RSBRK — COMMENTED** [review]
Why not set it as a draft PR?

File: api-errors.json [hunk 1]
```diff
@@ -195,6 +195,10 @@
         "short_message": "Invalid Barcode",
         "long_message": "The barcode could not be created due to invalid barcode type"
     },
+    "40099": {
```
Comment (zigge-zagge) [inline api-errors.json:198]: Use code 40053 and move it up. It was stupid to add the code 40098 a few years back. It incorrectly looks like we're almost out of numbers. Perhaps you could also leave a comment if you think that another person will do the same later on.

Comment (zigge-zagge) [reply api-errors.json:198]: Actually I think that 40049 to 40051 has not been used either 🤦 .

**Comment by zigge-zagge** [comment]
But why did you do [this one](https://github.com/kivra/kivra-api-errors/pull/18)? The error code 40923 may now be taken for another error since it's the clear successor to be picked. 40050 and 40051 are "protected" by later codes. These error codes are supposed to be universal if they are exposed to partners and senders. They are also documented publicly [here](https://developer.kivra.com/#section/Errors/Error-codes). Not saying this is all perfect though but in general we want one error code to only mean one thing.

**Comment by kivra-thohut** [comment]
> But why did you do [this one](https://github.com/kivra/kivra-api-errors/pull/18)? The error code 40923 may now be taken for another error since it's the clear successor to be picked. 40050 and 40051 are "protected" by later codes. These error codes are supposed to be universal if they are exposed to partners and senders. They are also documented publicly [here](https://developer.kivra.com/#section/Errors/Error-codes). Not saying this is all perfect though but in general we want one error code to only mean one thing.

This one was for removing errors that the forms service returns. This PR is for an error that kivra core returns.

**Pull request merged**